lib/mix/lib/mix/project.ex

```diff
@@ -378,19 +378,53 @@ defmodule Mix.Project do
   @doc """
   Returns the full path of all dependencies as a map.
 
+  ## Options
+
+    * `:depth` - only return deps to the depth level, a depth of 1
```

Comment:
Should this be `:level`?

Reply:
Yes, it should be consistent of course. What is the preferable option name of depth and level?

Reply:
According to the code and tests, it should be `:level`. However, I would go with `:depth` in this case :)

Reply:
I prefer `:depth` because we are talking about a dependency tree.